researchspace/researchspace
# Check it a selected item is already in the tabset

# Why

If a user open an item from clipboard ( or whatever action that will add an item to the frame ) and that item is already in the frame, a new tab will be added with the same item, leading to a multiple tabs of the same item.

# What

When a `Dashboard.AddFrame` event is triggered, before adding a the item to the frame check if that item is already present in the frame tabset.
If it is already there, then select the relevant item tab and set to active rather than add the same tab item again.

# Video / Gif / Screenshot


https://github.com/researchspace/researchspace/assets/6029465/2e20d3e8-4596-41bf-9b2e-4c5af9d5da90


# How To Test

Open same item from clipboard ( or simliar ) and check that the item is added to the frame only once and is selected
